include/seastar/rpc/rpc_impl.hh

```diff
@@ -451,6 +451,8 @@ relative_timeout_to_absolute(rpc_clock_type::duration relative) {
     return now + std::min(relative, rpc_clock_type::time_point::max() - now);
 }
 
+static constexpr size_t request_frame_headroom = 28;
```

Comment:
IIUC, 28 is `request_frame_with_timeout::raw_header_size`. maybe we can have a comment explaining the connection between 28 and `request_frame_with_timeout`?